functions-and-scope.js

```diff
@@ -72,6 +97,17 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+function highestGrade(array) {
+    let highestNumber = array[0];
```

Comment:
Top dat je hier de eerste waarde van de array hebt gepakt!!! Heel vaak zie ik dat hier `0` komt te staan, maar met deze soort operaties is het niet slim om dat te doen, omdat alle waardes in de array zelfs ónder de 0 kunnen zitten en dan heb je een foutief resultaat. Goed van je! 🏅